pegasus/sites.v3/code.org/views/learn_code_studio.haml

```diff
@@ -12,6 +12,10 @@
   else
     launch_path = "/starwars"
   end
+  if request.GET['company']
+    company = URI.escape(request.GET['company'])
+    response.set_cookie('company', {value: company, domain: ".code.org", path: '/', expires: Time.now + (365243600)})
```

Comment:
What is this mysterious number 365243600? Can we make it a const with a friendly name? Do rubyists do stuff like that?